ark/segmentation/marker_quantification.py

```diff
@@ -77,6 +78,8 @@ def compute_marker_counts(input_images, segmentation_masks, nuclear_counts=False
             nuc_mask = segmentation_utils.split_large_nuclei(cell_segmentation_mask=cell_mask,
                                                              nuc_segmentation_mask=nuc_mask,
                                                              cell_ids=unique_cell_ids)
+            nuc_mask = remove_small_objects(ar=nuc_mask, min_size=5)
```

Comment:
An alternative would be to put the call to `remove_small_objects` inside `split_large_nuclei` and add a parameter to `split_large_nuclei` to control the `min_size` parameter of `remove_small_objects`. If `min_size is not None`, then we make the additional call. This would be helpful if we plan on calling `remove_small_objects` every time after `split_large_nuclei` is called.